server/Managers/Events/AdminEvents.py

```diff
@@ -66,6 +67,9 @@ def delete(self):
                 return False
         return False
 
+    def get_event_data(self):
+        return self.event.covert_to_dict()
```

Comment:
the event id you got here is ObjectID, and you may want to stringify it to return as JSON object to client